ncm-useraccess/src/main/perl/useraccess.pod

```diff
@@ -233,37 +220,37 @@ We all know how evil Mr Burns is. So, let's say he wants full control
 on the Simpson family. And Homer wants to spy women at home:
 
  "/software/components/useraccess/roles/badburns" = nlist (
-	"kerberos4", list (nlist (
-		"realm", "SPRINGFIELD.COM",
-		"principal", "mrburns")),
-	"kerberos5", list (nlist (
-		"realm", "SPRINGFIELD.COM",
-		"principal", "mrburns")
-		)
+    "kerberos4", list (nlist (
+        "realm", "SPRINGFIELD.COM",
+        "principal", "mrburns")),
+    "kerberos5", list (nlist (
+        "realm", "SPRINGFIELD.COM",
+        "principal", "mrburns")
+        )
  );
 
  "/software/components/useraccess/roles/badhomer" = nlist (
-	"kerberos4", list (nlist (
-		"realm", "SPRINGFIELD.COM",
-		"principal", "homer",
-		"instance", "another_silly_project")),
-	"acls", list ("system-auth") # Woops! now Homer can't log-in!
-	);
+    "kerberos4", list (nlist (
+        "realm", "SPRINGFIELD.COM",
+        "principal", "homer",
+        "instance", "another_silly_project")),
+    "acls", list ("system-auth") # Woops! now Homer can't log-in!
+    );
 
  "/software/components/@COMPS@/users/marge/roles" = list (
```

Comment:
`@COMPS@` is not expanded anymore.  Can you please replace it with `useraccess` here?